src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -96,21 +110,38 @@ public static void init()
         registerDimension( 1, DimensionType.THE_END);
     }
 
-    public static void registerDimension(int id, DimensionType type)
+    /**
+     * Registers a dimension with a given delay
+     * @param id The ID of the dimension
+     * @param type The type of the dimension
+     * @param delayToUnload the delay in ticks the dimension should stay loaded after being queued to unload. Must not be negative
+     */
+    public static void registerDimension(int id, DimensionType type, int delayToUnload)
```

Comment:
Please remove modder-control of the `delayToUnload`, it should be only controlled by the person running the server. This will allow server admins to fix issues that are particular to their server.